src/scr_vm_functions.c

```diff
@@ -2995,3 +2995,78 @@ void PlayerCmd_SetSpectatedClient(scr_entref_t arg) {
 
 	gentity->client->sess.spectatorClient = Scr_GetInt(0);
 }
+
+void PlayerCmd_GetClientVersion(scr_entref_t arg) {
+	gentity_t* gentity;
+	int entityNum = 0;
+	mvabuf;
+
+	if (HIWORD(arg)) {
+		Scr_ObjectError("Not an entity");
+	} else {
+		entityNum = LOWORD(arg);
+		gentity = &g_entities[entityNum];
+
+		if (!gentity->client) {
+			Scr_ObjectError(va("Entity: %i is not a player", entityNum));
+		}
+	}
+
+    client_t *cl = &svs.clients[entityNum];
+	if (strlen(cl->xversion) < 2)
+		Scr_AddBool(0);
```

Comment:
I would change this to return `undefined`, which semantically makes a lot more sense than 0 as in reality the server just couldn't find the client version. Also the condition should be changed, you're looking for an empty `xversion` field, not one with less than 2 characters.

```c
if (strlen(cl->xversion) > 0) {
	Scr_AddString(cl->xversion);
} else {
	Scr_AddUndefined();
}
```